examples/mnist/tests/test_generate_mnist_dataset.py

```diff
@@ -18,7 +18,7 @@
 import sys
 import torch  # pylint: disable=unused-import
 
-import examples.mnist.main as main
+import examples.mnist.pytorch_example as pytorch_example
```

Comment:
Thank you for this rename, @kashmawy.

One more thing, did you make sure to run the test to ensure passing?\
1. `cd examples/mnist/tests`
2. `PYTHONPATH=<path_to_repo> pytest -v`